This program is not very useful as a downstream, it seems to be only intended
for internal QA. It also has a lot of weird things like hardcoding the prefixed
cpp to use and putting a text file in /usr/bin.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,7 +5,6 @@
 # List of subsidiary CMakeLists
 add_subdirectory(dtmerge)
 add_subdirectory(otpset)
-add_subdirectory(overlaycheck)
 add_subdirectory(ovmerge)
 add_subdirectory(pinctrl)
 add_subdirectory(raspinfo)
